core/src/main/java/org/frankframework/statistics/HasApplicationContext.java

```diff
@@ -0,0 +1,38 @@
+/*
+   Copyright 2013 Nationale-Nederlanden, 2022-2025 WeAreFrank!
```

Comment:
Dit wordt gezien als nieuwe file, dus hoeft alleen copyright 2025 (Maar wanneer deze weer in het correcte package staat dan komt de history misschien weer terug)?